packages/atomic/create-atomic/src/plopfile.ts

```diff
@@ -92,18 +93,18 @@ export default async function (plop: NodePlopAPI) {
         message:
           'Use an existing hosted search page as a template, or start from scratch?',
         choices: async function (answers: PlopData) {
-          initPlatformClient(answers);
           return [
             {value: undefined, name: 'Start from scratch'},
             {type: 'separator'},
-            ...(await listSearchPagesOptions(platformClient)),
+            ...(await listSearchPagesOptions(initPlatformClient(answers))),
           ];
         },
         validate: async (input, answers: PlopData) => {
```

Comment:
Woops, I did a refactor to always fetch the page on the validate method. To my surprise, there is no validation when picking from the list.